Donell/exceptions_challenge.py

```diff
@@ -0,0 +1,71 @@
+
+# Question 1: validate_user_input
+
+from xml.dom import ValidationErr
+
+
+def validate_user_input():
+    user_number = validate_user_input
+    user_number(input("please enter a number"))
```

Comment:
The logic here is incorrect. We are trying to assign the input from the user into user_number variable, so the correct syntax should be : user_input = input("Please enter a number").